### ISSUE TITLE
track actual as well as possible

### ISSUE THREAD

**Issue by nigelmegitt** [w3c/spec-releases#2]
I just tried this out for TTML2 to see what it would do.

In the first box I entered the FPWD date of `2015-02-12`, which filled in the earliest possible next dates. Then I entered a predicted CR date of `2017-10-30`, which the tool kindly rejected and replaced with the first available date following, which is 2017-10-31, however it then backwards-updated the FPWD to 2017-08-01. It would be nice to be able to lock a date in the past to say "no, this is the actual date when it really happened" and have the tool then tell you the status of the exclusion opportunities etc, i.e. "expired" or "open".

**Comment by plehegar** [comment]
Indeed, the tool doesn't take into account actual data from our /TR database. Adding this one on the TODO pile.

**Comment by nigelmegitt** [comment]
Even if you don't integrate with the /TR database (which would be cool), allowing the dates to be manually entered would be a start.

**Comment by plehegar** [comment]
thinking out aloud: I need to only compute the dates that haven't been entered by the user or in the url.
